.github/describe_supported_versions_diff.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu
+
+readonly SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
+readonly ROOT_DIR="$(dirname ${SCRIPT_DIR})"
+
+echo "feat: Test additional package versions [auto-test-update]"
+
+git diff --name-only -- "${ROOT_DIR}/instrumentation/" | \
+    sort | \
+    while read -r modified_version_file; do \
```

Comment:
maybe this line should be described a bit/ documented? to be more readable

Reply:
Added a comment